spring-reactive-modules/spring-5-reactive-client-2/src/main/java/com/baeldung/limitrequests/client/LimitConcurrency.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.limitrequests.client;
+
+import org.springframework.web.reactive.function.client.WebClient;
+
+import com.baeldung.limitrequests.client.utils.Client;
+import com.baeldung.limitrequests.client.utils.RandomConsumer;
+
+import reactor.core.publisher.Flux;
+
+public class LimitConcurrency {
+
+    private LimitConcurrency() {
+    }
+
+    public static Flux<Integer> fetch(WebClient client, int requests, int concurrency) {
+        String clientId = Client.id(requests, LimitConcurrency.class.getSimpleName(), concurrency);
+
+        return Flux.range(1, requests)
+            .log()
+            .flatMap(i -> {
+                return RandomConsumer.get(client, clientId);
+            }, concurrency);
```

Comment:
Same here, it can be simplified to: 
`            .flatMap(i -> RandomConsumer.get(client, clientId), concurrency);`